pkgs/development/python-modules/plumbum/default.nix

```diff
@@ -1,19 +1,64 @@
-{ buildPythonPackage
-, fetchPypi
-, pytest
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, paramiko
+, psutil
+, pytest-cov
+, pytest-mock
+, setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "plumbum";
   version = "1.7.0";
 
-  checkInputs = [ pytest ];
+  # pypi doesn't contain version information for package
+  src = fetchFromGitHub {
+    owner = "tomerfiliba";
+    repo = "plumbum";
+    rev = "v${version}";
+    sha256 = "sha256-ATEInozJTmzNbLDDjiWMnA5IgVBG2hmKVyw/vsiPiu4=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  checkInputs = [
+    paramiko
+    psutil
+    pytest-cov
```

Comment:
```suggestion
```
collecting coverage data has no benefit for us or upstream except that it takes ressources.